planetiler-core/src/main/java/com/onthegomap/planetiler/Planetiler.java

```diff
@@ -793,10 +801,15 @@ public void run() {
     ensureInputFilesExist();
 
     if (fetchWikidata) {
-      Wikidata.fetch(osmInputFile(), wikidataNamesFile, config(), profile(), stats());
+      Wikidata.fetch(osmInputFile(), wikidataNamesFile, config(), profile(), stats(), wikidataMaxAge,
+        wikidataUpdateLimit);
     }
     if (useWikidata) {
-      translations().addFallbackTranslationProvider(Wikidata.load(wikidataNamesFile));
+      // update of old items handled by "fetch" above => do NOT use age checks here
+      var translationsProvider = Wikidata.load(wikidataNamesFile, Duration.ZERO, 0);
+      translationsProvider.clearUpdateTimes();
+      translations()
+        .addFallbackTranslationProvider(translationsProvider);
```

Comment:
Can you add another `Wikidata.load` method that retains the old behavior so this if block can just remain the same?  The normal behavior I'd expect calling "load" externally is "load everything" so maybe the "load with time limit" method should just be private/internal since it's an implementation detail of the "load for update" external API?

Reply:
Adjusted.